src/app/containers/Timestamp/timestampUtilities.js

```diff
@@ -14,3 +15,13 @@ export const formatUnixTimestamp = (timestamp, momentString, timezone) =>
   moment(timestamp)
     .tz(timezone)
     .format(momentString);
+
+export const showRelativeTime = (timestamp, isRelative, format, timezone) => {
+  if (isRelative) {
+    return relativeTime(timestamp);
+  }
+  if (!isRelative && format) {
+    return formatUnixTimestamp(timestamp, format, timezone);
+  }
+  return formatUnixTimestamp(timestamp, 'D MMMM YYYY, HH:mm z', timezone);
```

Comment:
Not sure if I like the default being set here. Curious of others thoughts, at the least it can be declared as `const defaultFormat` at the top of this function :P

Reply:
Declared a `defaultFormat` to hold the default.